<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1262</width>
    <height>979</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QFrame" name="frame">
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>database type</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_db_type"/>
          </item>
          <item>
           <widget class="QLabel" name="label_13">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>FZKai-Z03S</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>database list</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_db_list"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_load">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>FZKai-Z03S</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>load</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>FZKai-Z03S</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>current DB info</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="splitter_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QPlainTextEdit" name="plainTextEdit_project_info">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50000</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="plainText">
            <string>database info goes here</string>
           </property>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_23">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_update_project_info">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>FZKai-Z03S</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>update</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Terminal</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="ConsoleWidget" name="widget_terminal" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Collection Info</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QFrame" name="frame_4">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <layout class="QFormLayout" name="formLayout">
                 <property name="sizeConstraint">
                  <enum>QLayout::SetNoConstraint</enum>
                 </property>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Database operations</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_add">
             <property name="text">
              <string>add</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_update">
             <property name="text">
              <string>update</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_delete">
             <property name="text">
              <string>delete</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frame_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>5000</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Document list</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tableView_2">
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>search DB</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>field 1</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QComboBox" name="comboBox_2">
             <item>
              <property name="text">
               <string>and</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>or</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>not</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QPushButton" name="pushButton_6">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
           <item row="0" column="4">
            <widget class="QLineEdit" name="lineEdit_2"/>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>field 2</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1262</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDatabaseCloud"/>
   <addaction name="actionRegistration"/>
   <addaction name="separator"/>
   <addaction name="actionLogout"/>
   <addaction name="actionLocalServerOn"/>
   <addaction name="actionDatabaseInfo"/>
   <addaction name="separator"/>
   <addaction name="actionSearchDB"/>
   <addaction name="actionImportBiTex"/>
   <addaction name="actionExportBiTex"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionExportBiTex">
   <property name="icon">
    <iconset>
     <normaloff>icons/export.png</normaloff>icons/export.png</iconset>
   </property>
   <property name="text">
    <string>ExportBiTex</string>
   </property>
   <property name="toolTip">
    <string>export paper to BiTex file</string>
   </property>
  </action>
  <action name="actionDatabaseCloud">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../pyworkship/pygod/apps/py_scheduler/resources/icons/cloud.png</normaloff>../../../../../pyworkship/pygod/apps/py_scheduler/resources/icons/cloud.png</iconset>
   </property>
   <property name="text">
    <string>DatabaseCloud</string>
   </property>
   <property name="toolTip">
    <string>登录系统</string>
   </property>
  </action>
  <action name="actionImportBiTex">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../pyworkship/pygod/apps/py_scheduler/resources/icons/import.png</normaloff>../../../../../pyworkship/pygod/apps/py_scheduler/resources/icons/import.png</iconset>
   </property>
   <property name="text">
    <string>ImportBiTex</string>
   </property>
   <property name="toolTip">
    <string>Extract paper info from BiTex file</string>
   </property>
  </action>
  <action name="actionLocalServerOn">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../pyworkship/pygod/apps/py_scheduler/resources/icons/serveron.png</normaloff>../../../../../pyworkship/pygod/apps/py_scheduler/resources/icons/serveron.png</iconset>
   </property>
   <property name="text">
    <string>LocalServerOn</string>
   </property>
   <property name="toolTip">
    <string>start local mongo server</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../pyworkship/pygod/apps/py_scheduler/resources/icons/serveroff.png</normaloff>../../../../../pyworkship/pygod/apps/py_scheduler/resources/icons/serveroff.png</iconset>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
   <property name="toolTip">
    <string>注销当前用户</string>
   </property>
  </action>
  <action name="actionRegistration">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../pyworkship/pygod/apps/py_scheduler/resources/icons/client.png</normaloff>../../../../../pyworkship/pygod/apps/py_scheduler/resources/icons/client.png</iconset>
   </property>
   <property name="text">
    <string>Registration</string>
   </property>
   <property name="toolTip">
    <string>注册新用户</string>
   </property>
  </action>
  <action name="actionDatabaseInfo">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../pyworkship/pygod/apps/py_scheduler/resources/icons/info.png</normaloff>../../../../../pyworkship/pygod/apps/py_scheduler/resources/icons/info.png</iconset>
   </property>
   <property name="text">
    <string>DatabaseInfo</string>
   </property>
   <property name="toolTip">
    <string>get database information</string>
   </property>
  </action>
  <action name="actionSearchDB">
   <property name="icon">
    <iconset>
     <normaloff>../../../../../pyworkship/pygod/apps/py_scheduler/resources/icons/search.png</normaloff>../../../../../pyworkship/pygod/apps/py_scheduler/resources/icons/search.png</iconset>
   </property>
   <property name="text">
    <string>SearchDB</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConsoleWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph.console.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
